lib/utils/child-process.js

```diff
@@ -14,7 +14,7 @@ process.on( 'message', onMessage );
  * @param {Options} data.options The options object.
  */
 function onMessage( data ) {
-	const log = require( './log' )();
+	const log = require( './getlogger' )();
```

Comment:
The variable name is ok. The module name is ok too. Note that the getLogger() function is immediately executed.